apps/dapp/src/components/Pages/Core/DappPages/Dashboard/Chart/V2StrategyMetricsChart.tsx

```diff
@@ -73,17 +73,21 @@ const V2StrategyMetricsChart: React.FC<{
   selectedInterval: ChartSupportedTimeInterval;
 }> = ({ dashboardType, selectedMetric, selectedInterval, strategyNames }) => {
   // uncamel-case the metric names
-  const formatMetricName = (name: string) =>
+  const formatMetricName = (name: string) => {
+
     // format only the selected metric name (the selected metric or all lines in a TRV chart)
-    name === selectedMetric || dashboardType === DashboardType.TREASURY_RESERVES_VAULT
-      ? name
+    if (name === 'debtUSD') return 'Debt (USD)';
```

Comment:
A bit gross why does it come through like this?

Reply:
I haven't dug into this code, but is it possible to pass through a formatter lambda at runtime which says how to format that particular graph?

Reply:
Can approve for now, but yeh it's not very scalable